### PR TITLE
Update manage-addins-in-the-admin-center.md

### PR DESCRIPTION
**Pull request by msbemba**
I have added the note based on the internal ADO item and discussion - https://office.visualstudio.com/OC/_workitems/edit/5121015?src=WorkItemMention&src-action=artifact_link

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5302

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 1]
```diff
@@ -116,6 +116,9 @@ Support for turning off the Office Store is available in the following versions:
 - The web - Currently available.
     
 This does not prevent an administrator from using Centralized Deployment to assign an add-in from the Office Store.
+
+> [!NOTE] 
+> Add-ins such as Visio Data Visualizer, Bing Maps, and People Graph will still show up in the ribbon, even if an admin has disabled the Store. To remove these links, adminisrators must disable the store through Group Policy Object (GPO).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
> Add-ins such as Visio Data Visualizer, Bing Maps, and People Graph will still show up in the ribbon, even if an admin has disabled the Store. To remove these links, administrators must disable the Store through Group Policy Object (GPO).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by AlexJerabek** [comment]
Pinging this thread to see when this PR will be merged. There's a related customer issue we'd like to close out.

Thanks!

**Comment by yogkumgit** [comment]
Gentle reminder @kwekuako

**Pull request merged**